lib/velum/secrets.rb

```diff
@@ -0,0 +1,20 @@
+require 'securerandom'
+
+module Velum
```

Comment:
I'd create either a `Secrets` module or class inside of the `Velum` module. That's what a ruby developer would expect I think.